docs/theory/diffusion/diffusion.rst

```diff
@@ -0,0 +1,667 @@
+Diffusion analysis
+==================
+
+.. only:: html
+
+    :nbexport:`Download this page as a Jupyter notebook <self>`
+
+Pylake supports a number of methods for studying diffusive processes.
+These methods rely on quantifying the movement of the fluorophore.
+This section details some of the available methods and details some pitfalls with each of them.
+In this chapter, we will be making use of simulated data, so that we can study the quality of our estimates while knowing the ground truth.
+Note that the code to perform these simulation is included, so that you can play with them and produce similar graphs which are more tailored to your own experimental conditions.
```

Comment:
Awesome that you included the simulation code!

Reply:
Idea was to have fully runnable examples. I hope to do the same one day for force calibration. I hope it might empower those who are comfortable with Python to do numerical experiments for their own specific situations, thereby having more eyes on the code and maybe identify additional corner cases we did not foresee.